### ISSUE TITLE
Vault commands slow when using the load balancer address.

### ISSUE THREAD

**Issue by robertdebock** [robertdebock/terraform-aws-vault#18]
Vault commands take some 5 - 15 seconds to process. This eventually works, but is annoying.

Any `vault` command results in:

```text
uname({sysname="Linux", nodename="ip-172-16-127-35.eu-west-1.compute.internal", ...}) = 0
epoll_pwait(4, [], 128, 0, NULL, 138566752) = 0
futex(0x4000094550, FUTEX_WAKE_PRIVATE, 1) = 1
```

> NOTE: The `nodename` is set to `ip-172-16-127-35.eu-west-1.compute.internal`. That's not correct, should be the external address. (api_address).

From now on a loop starts: (Likely causing the delay.)

```text
--- SIGURG {si_signo=SIGURG, si_code=SI_TKILL, si_pid=1797, si_uid=0} ---
rt_sigreturn({mask=[]})                 = 1
```

The [`api_addr`](https://www.vaultproject.io/docs/configuration#api_addr) is set correctly in `/etc/vault.d/vault.hcl`:

```hcl
api_addr          = "https://dev.robertdebock.nl:8200"
```

Maybe the `node_id` causes this behaviour:

```hcl
storage "raft" {
  path    = "/opt/vault/data"
  node_id = "ip-172-16-127-35.eu-west-1.compute.internal"
```

**Comment by robertdebock** [comment]
The load balancer returns 3 IP-addresses, 2 of these do not pass TCP port 8200:


Works:

```shell
robertdb@MacBook-Pro-van-Robert terraform-aws-vault % telnet 34.247.75.149 8200
Trying 34.247.75.149...
Connected to ec2-34-247-75-149.eu-west-1.compute.amazonaws.com.
Escape character is '^]'.
^]
telnet> c
Connection closed.
```

Broken:

```shell
robertdb@MacBook-Pro-van-Robert terraform-aws-vault % telnet 54.77.117.228 8200
Trying 54.77.117.228...
telnet: connect to address 54.77.117.228: Operation timed out
telnet: Unable to connect to remote host
```

Broken:
```shell
robertdb@MacBook-Pro-van-Robert terraform-aws-vault % telnet 52.50.149.226 8200
Trying 52.50.149.226...
telnet: connect to address 52.50.149.226: Operation timed out
telnet: Unable to connect to remote host
```

**Comment by robertdebock** [comment]
Only one subnet has a route set to the internet gateway.

**Comment by robertdebock** [comment]
Fixed in release 3.0.1.
